### ISSUE TITLE
FYI: SPDX is Working on Structured-Data License Text Forms

### ISSUE THREAD

**Issue by kemitchell** [sindresorhus/spdx-license-list#10]
The SPDX working groups have put a lot of effort into translating the license text forms into a machine-readable template format. That work is here:

https://github.com/spdx/license-list-XML

They have not yet blessed the XML repository as "ready" or "official". But progress has been very good.  If and when it's ready, you may prefer to consume that source for license texts.

cc @jlovejoy @bradleeedmondson, with a quick note: This is one of a few packages in the Node.js/JavaScript ecosystem that packages up and publishes the license list as a data library. Most of these use scripts to pull down JSON or other files from spdx.org.  This package pulls down license texts in addition to list metadata. [spdx-license-ids](https://github.com/shinnn/spdx-license-ids) is just license IDs.

**Comment by kemitchell** [comment]
When the time comes, I may be the one who actually ends up figuring out the SPDX template XML format and coming up with the code to translate to JSON.  I've already done JSON templates for the most popular licenses under https://github.com/jslicense.  Here's MIT, for example: https://github.com/jslicense/jslicense-mit/blob/master/index.json

We obviously don't need more than one effort copying out and proofreading license templates!

**Comment by bradleeedmondson** [comment]
FWIW I believe the thought from the SPDX side is that once our XML format is stable and the data is ready, we would use that XML format to continue to produce and release the other formats of the license list (RDF, etc.) which could be expanded to include JSON. That is, I'm not sure the idea is to have the XML ever be directly consumed.

I'm only mildly technical though; I believe @goneall would be able to confirm or correct. Gary, is that right? And if so, could/might it change if people want to consume the XML?

**Comment by goneall** [comment]
The JSON formats are available on the website and on github.  The github url is https://github.com/spdx/license-list-data/tree/master/json and the web url is https://spdx.org/licenses/licenses.json for the index and https://spdx.org/licenses/[spdx-id].json where [spdx-id] is the SPDX license short identifier.

I have on my todo list to better document this better.

The XML format is intended to be internal to the SPDX legal team and is subject to change.  That being said, we did try to align the format with the standard to minimize potential future incompatibilities.
